Some Docs fixes (#320)
RedisGears/RedisGears: docs/operations.md

## AggregateBy
The sugar **AggregateBy** operation performs many-to-less mapping (N:M) of records.

It is similar to the [Aggregate](#aggregate) operation but aggregates per key. It requires a an [extractor](#extractor) callback, a zero value and two [reducers](#reducer) callbacks for computing the local and global aggregates.

The operation is made of these steps:

  1. extraction of the groups using [extractor](#extractor)
  1. The local [reducer](#reducer) is executed locally and initialized with the zero value
  1. A global [repartition](#repartition) operation that uses the [extractor](#extractor)

>   1. The global [reducer](#reducer) is executed on each shards when we know values are located correctly on the cluster

Reviewer: ```suggestion
  1. The global [reducer](#reducer) is executed on each shard once it is repartitioned with its relevant
```
